services/core-web/src/components/noticeOfWork/applications/NOWDocuments.js

```diff
@@ -188,7 +306,20 @@ export const NOWDocuments = (props) => {
       value: item.value,
     }));
 
-    const fileNameColumn = props.selectedRows
+    const sortColumn = {
+      title: "Order",
+      dataIndex: "index",
+      className: "drag-visible",
+      render: (text) => (
+        <>
+          <DragHandle />
+          {/* NOTE: We are adding 2 here because "1.1" in the issued permits is currently always the application form document. */}
+          &nbsp; 1.{text + 2}
+        </>
```

Comment:
order starts at 0?

Reply:
We use the index (starting from 0) and then need to add 2 to get "1.2".